src/herder/Herder.h

```diff
@@ -67,8 +67,14 @@ class Herder
 
     enum State
     {
+        // Starting up, no state is known
```

Comment:
you added new states but forgot to update the string array - I opened PR https://github.com/stellar/stellar-core/pull/3078 to catch this at compile time as this can crash the app

Reply:
Good catch, this is fixed now